### PR TITLE
default value of context as empty string rather than empty array

### PR DESCRIPTION
**Pull request by gavinerickson**

### CODE REVIEW

**Comment by gavinerickson** [comment]
Thanks for the tip, have corrected

File: Controller/LogController.php [hunk 1]
```diff
@@ -20,7 +20,7 @@ public function createAction(Request $request)
         if(Request::METHOD_GET === $request->getMethod()){
             $level = (string) $request->query->get('level');
             $message = (string) $request->query->get('msg');
-            $context = (array) $request->query->get('context', array());
+            $context = (array) $request->query->all('context', array());
```
Comment (stof) [inline Controller/LogController.php:23]: the code should be simplified:

- `all` does not have a second argument (the default value is always an array
- the array cast is useless

**Review by stof — CHANGES_REQUESTED** [review]
This change requires bumping the min version of symfony/http-foundation to 5.3, which is the first version support calling `all` with an argument.

**Comment by Seldaek** [comment]
Thanks, this was fixed by #47 tho so closing

**Pull request closed without merging**